Dockerfile

```
@@ -17,9 +17,9 @@ RUN useradd -m appuser && \
 USER appuser
 
 # Install Python dependencies
-COPY requirements.txt .
+COPY pyproject.toml .
 RUN pip install --no-cache-dir --upgrade pip && \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)